homeassistant/components/verisure/switch.py

```diff
@@ -11,7 +11,7 @@
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
-from .const import CONF_GIID, DOMAIN
+from .const import CONF_GIID, DOMAIN, LOGGER
```

Comment:
CI is failing on unused import here.
* https://github.com/home-assistant/core/actions/runs/4491557553/jobs/7916676718?pr=89318
* https://github.com/home-assistant/core/actions/runs/4491557553/jobs/7916538012?pr=89318

The following change would fix that:

```suggestion
from .const import CONF_GIID, DOMAIN
```